build/MSBuildExtensions.targets

```diff
@@ -99,7 +99,7 @@
     <ItemGroup>
       <_NETStandardLibraryVersions Include="@(PackageDefinitions->'%(Version)')"
                                    Condition="%(PackageDefinitions.Name) == 'NetStandard.Library'" />
-      <_NETCorePlatformsImplicitPackageVersion Include="@(PackageDefinitions->'%(Version)')"
+      <_NETCorePlatformsLibraryVersions Include="@(PackageDefinitions->'%(Version)')"
                                    Condition="%(PackageDefinitions.Name) == 'Microsoft.NETCore.Platforms'" />
     </ItemGroup>
```

Comment:
Below here: https://github.com/dotnet/cli/blob/02a98d4e63e02928b8d359e4a0fff7d78530b67f/build/MSBuildExtensions.targets#L106-L107

There should also be an error if we do not find exactly one version of Microsoft.NETCore.Platforms

Reply:
Fixed